node/full_node_test.go

```diff
@@ -303,6 +363,46 @@ func createAggregatorWithPersistence(ctx context.Context, dbPath string, dalc *d
 	return fullNode, app
 }
 
+func createAggregatorWithApp(ctx context.Context, app abci.Application, voteExtensionEnableHeight int64, t *testing.T) (Node, crypto.PrivKey) {
```

Comment:
I don't think this name is great. It also feels like a lot of overlap with `createAggregatorWithPersistence`.

Can we, as a follow up maybe, clean these up and deduplicate?